java.base/src/main/java/java/lang/Thread$_patch.java

```diff
@@ -231,6 +233,9 @@ void unblocked() {
     @Add
     private static volatile int nonDaemonThreadCount;
 
+    @Add
+    private static volatile int shutdownInitiated;
```

Comment:
logically this should be a boolean, but I was having problems getting compareAndSet on a boolean field to compile and this was expedient... can fix the compiler issues and then change this to be a boolean later...